post-install/sleep.md

```diff
@@ -0,0 +1,272 @@
+
+* See here for more info:
+    * [Disabling desktop dGPUs](https://dortania.github.io/Getting-Started-With-ACPI/Desktops/desktop-disable.md)
+    * [Disabling laptop dGPUs](https://dortania.github.io/Getting-Started-With-ACPI/Laptops/laptop-disable.md)
+	
+Special notes for iGPU users on 10.15.4 and newer:
+
+* iGPU wake is partially broken due to numerous hacks apple uses in AppleGraphicsPowerManagement.kext with real Macs, to get around this you'll likely need `igfxonln=1` to force all displays online. Obviously test first to make sure you have this issue.
+* AAPL,ig-platform-id `07009B3E` may fail for desktop Coffee Lake (UHD 630) users, you can try `00009B3E` instead
+
+Special note for 4k Displays with AMD dGPUs:
+
+* Some displays may fail to wake randomly, mainly caused by AGDC preferences. To fix, apply this to your dGPU in DeviceProperties:
+  * `CFG,CFG_USE_AGDC | Data | 00`
+  * You can find the PciRoot of your GPU with [gfxutil](https://github.com/acidanthera/gfxutil/releases)
+    * `/path/to/gfxutil -f GFX0`
+
+![](/images/post-install/sleep-md/agdc.png)
+
+### Fixing Thunderbolt
+
+Thunderbolt is a very tricky topic in the community, mainly due to the complexity of the situation. You really have just 2 paths to go down if you want Thunderbolt and sleep to work simultaneously:
+
+* Disable Thunderbolt 3 in the BIOS
+* Attempt to patch Thunderbolt 3:
+  * [Thunderbolt 3 Fix](https://osy.gitbook.io/hac-mini-guide/details/thunderbolt-3-fix/)
+  * [ThunderboltReset](https://github.com/osy86/ThunderboltReset)
+  * [ThunderboltPkg](https://github.com/al3xtjames/ThunderboltPkg/blob/master/Docs/FAQ.md)
+
+Note: Thunderbolt can be enabled without extra work *if* you're ok without sleep, and vice versa.
+
+### Fixing NICs
+
+NICs(network Interface Controllers) are fairly easy to fix with sleep, it's mainly the following:
+
+* Disable `WakeOnLAN` in the BIOS
+  * Most hack kexts don't support WOL
+* Disable `Wake for network access` in macOS(SystemPreferences -> Power)
+  * Seems to break on a lot of hacks
+  
+  
+### Fixing NVMe
+
+So macOS can be quite picky when it comes to NVMe drives, and there's also the issue that Apple's power management drivers are limited to Apple branded drives only. So the main things to do are:
+
+* Make sure the NVMe is on the latest firmware(especially important for [970 Evo Plus drives](https://www.tonymacx86.com/threads/do-the-samsung-970-evo-plus-drives-work-new-firmware-available-2b2qexm7.270757/page-14#post-1960453))
+* Use [NVMeFix.kext](https://github.com/acidanthera/NVMeFix/releases) to allow for proper NVMe power management
+
+And avoid problematic drives, the main culprits:
+
+* Samsung's PM981 SSDs
```

Comment:
You can also add PM991, it has the same behavior as PM981 (check the discord server)